wsidicomizer/common.py

```diff
@@ -109,7 +113,9 @@ def create_instance_dataset(
                 DSfloat(self.pixel_spacing.height, True)
             ]
             pixel_measure_sequence.SpacingBetweenSlices = 0.0
-            pixel_measure_sequence.SliceThickness = 0.0
+            # DICOM 2022a part 3 IODs - C.8.12.4.1.2 Imaged Volume Width, Height, Depth
+            # Depth must not be 0. Default to 0.5 microns
+            pixel_measure_sequence.SliceThickness = 0.0005
```

Comment:
Is 0.5 um a reasonable depth of field for a scanner?

Reply:
Honestly, I don't know. Looking at the very limited info in Hamamatsu's marketing (I only have Hamamatsu NDPI data), 0.5 um is possible as the distance between adjacent slices. From my MR background I assume that the slice thickness is the distance between slices or less. Since it cannot be zero according to the standard, this seemed a reasonable default until a better value is found, possibly in the TIFF header.

Reply:
User-specified DOF should be implemented in the future.